conans/test/integration/basic_build_test.py

```diff
@@ -50,16 +25,30 @@ def build_default_test(self):
 
         for cmd, lang, static, pure_c in [("install", 0, True, True),
                                           ("install -o language=1 -o static=False -g txt", 1, False, False)]:
-            self._build(cmd, static, pure_c, use_cmake=False, lang=lang)
-
-    def build_mingw_test(self):
-        if platform.system() != "Windows":
-            return
-        not_env = os.system("c++ --version > nul")
-        if not_env != 0:
-            logger.error("This platform does not support G++ command")
-            return
-        install = "install -s compiler=gcc -s compiler.libcxx=libstdc++ -s compiler.version=4.9"
-        for cmd, lang, static, pure_c in [(install, 0, True, True),
-                                          (install + " -o language=1 -o static=False", 1, False, False)]:
-            self._build(cmd, static, pure_c, use_cmake=False, lang=lang)
+            build(self, cmd, static, pure_c, use_cmake=False, lang=lang)
+
+
+def build(tester, cmd, static, pure_c, use_cmake, lang):
```

Comment:
Why taking this outside of the class?

Reply:
It's used in the `conan-io/test` to build with mingw